Jack/Script.fsx

```diff
@@ -78,6 +78,22 @@ Property.check <| forAll {
 }
 
 //
+// Conditional Properties
+//
+
+fun (x, y) -> (x > 0 && y > 0) ==> (x * y > 0)
+|> Property.forAll (Gen.zip Gen.int Gen.int)
+|> Property.check
```

Comment:
When I execute this in F# Interactive, I'm getting back:

```
*** Failed! Falsifiable (after 45 tests and 29 shrinks):
(879, 2443099)
val it : bool = false
> 
```

Most likely I did something wrong, otherwise it can be a bug (?) :confused: